spec/data-model/message.json

```diff
@@ -103,13 +109,43 @@
       ]
     },
 
+    "markup-open": {
+      "type": "object",
+      "properties": {
+        "type": { "const": "markup" },
+        "kind": { "enum": ["open", "standalone"] },
+        "name": { "type": "string" },
+        "options": { "$ref": "#/$defs/options" }
+      },
+      "required": ["type", "kind", "name"]
+    },
```

Comment:
why isn't this two objects, `markup-open` and `markup-standalone`, like the data model has?

See previous comment about `name` vs. `identifier`

Reply:
I merged them here because for the JSON Schema definition it's an internal detail that doesn't show up in user data, and the two shapes are otherwise exactly the same.

Would you be more comfortable if the whole object was duplicated?

Reply:
Yes: the open and standalone items are meant to be semantically different. And they wouldn't be exactly duplicated. One would have `kind: open` and the other `kind: standalone`

Reply:
Added.